packages/compass-saved-aggregations-queries/src/components/aggregations-queries-list.tsx

```diff
@@ -139,17 +155,7 @@ const AggregationsQueriesList = ({
         classNames={{ row: rowStyles }}
       ></VirtualGrid>
       <OpenItemModal></OpenItemModal>
-      {deletingItem && (
-        <DeleteItemModal
-          isOpen={true}
```

Comment:
That's something I missed during my second review for https://github.com/mongodb-js/compass/pull/2782, sorry for that @mabaasit 

When I suggested to pass isOpen as a prop, the important part there was that the Modal component itself receives the prop dynamically instead of being re-mounted completely, to address this I had to normalize the state here and move more related parts to redux store instead of using React state. Hope the explanation and comments I left of the refactor make sense. I'm also happy to chat about it if you feel like it